.travis.yml

```diff
@@ -1,4 +1,5 @@
-sudo: false
+dist: trusty
+sudo: required
```

Comment:
OK, after reading the blog post again, `sudo: required` will become the default in September, but only for `precise`, so I'll revert this part.